appknox/client.py

```diff
@@ -430,9 +430,21 @@ def upload_file(self, file_data: str):
                 url=response['url']
             )
         )
+        submission_id = response2['submission_id']
+        return self.get_file_from_submission_id(submission_id)
+
+    def get_file_from_submission_id(self, submission_id: int):
```

Comment:
I think better would be to name it something like `poll_submission_file` since rest of the `get_*` methods are merely doing direct api `GET`s.